src/app/metadata/models/dataset-information.ts

```diff
@@ -11,9 +11,10 @@ export interface DatasetInformation {
 
 export interface FileInformation {
   accession: string;
-  size: number;
-  sha256_hash: string;
-  storage_alias: string;
+  // the remaining fields may be undefined
```

Comment:
This doesn't seem sound at all, but I assume this is an issue with the backend (and is unrelated to this PR)?

Reply:
Yes, the backend sometimes doesn't return this fields in the response. This is not a bug, intended behavior and defined in the API. By marking the fields as undefined here, we get a TypeScript warning when using them unshielded, so we don't create the issues we had in the legacy app.

Reply:
Added explanation in comment now